Data/Base/Shaders/Common/Lighting.h

```diff
@@ -74,7 +74,7 @@ float3 SampleSceneColor(float2 screenPosition)
 
 float SampleSceneDepth(float2 screenPosition)
 {
-  float depthFromZBuffer = SceneDepth.SampleLevel(PointClampSampler, float3(screenPosition.xy * ViewportSize.zw, s_ActiveCameraEyeIndex), 0.0f).r;
```

Comment:
Since we are using a point filter we can switch to Load instead of Sample, but there are a few things that make Load tricky:
- Load takes absolute pixel coordinates instead of relative coordinates like Sample, so your change is wrong.
- We need to manually take care of clamping in the shader code.

Reply:
The absolute coordinates explain some artifacts I've been seeing, good to know.

Reply:
Added clamping and should now sample at the correct coordinate.